apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/CloudTrace.cs

```diff
@@ -126,12 +129,19 @@ private CloudTrace(string projectId, TraceConfiguration config = null, TraceServ
         ///     detected from the platform.</param>
         /// <param name="config">Optional trace configuration, if unset the default will be used.</param>
         /// <param name="client">Optional trace client, if unset the default will be used.</param>
-        public static void Initialize(HttpApplication application, string projectId = null, TraceConfiguration config = null, TraceServiceClient client = null)
+        /// <param name="traceOverridePredicate">Optional function to trace requests. If the trace header is not set
+        ///     then this function will be called to determine if a given request should be traced.  This will
+        ///     not override trace headers. If the function returns true the request will be traced, if false
+        ///     is returned the trace will not be traced and if null is returned it will not affect the
+        ///     trace decision.</param>
+        public static void Initialize(HttpApplication application, string projectId = null,
+            TraceConfiguration config = null, TraceServiceClient client = null,
+            Func<HttpRequest, bool?> traceOverridePredicate = null)
```

Comment:
Given how it is used `...Override...` seems a bit misleading. Perhaps `traceFallbackPredicate` would be better.